packages/aws-cdk-lib/aws-lambda/README.md

```diff
@@ -219,6 +219,68 @@ new lambda.Function(this, 'Lambda', {
 
 To use `applicationLogLevelV2` and/or `systemLogLevelV2` you must set `loggingFormat` to `LoggingFormat.JSON`.
 
+## Customizing Log Group Creation
+
+### Log Group Creation Methods
+
+| Method | Description | Tag Propagation | Prop Injection | Aspects | Notes |
+|--------|-------------|-----------------|----------------|---------|-------|
+| **logRetention prop** | Legacy approach using Custom Resource | False | False | False | Does not support TPA |
+| **logGroup prop** | Explicitly supplied by user in CDK app | True | True | True | Full support for TPA |
+| **Lazy creation** | Lambda service creates logGroup on first invocation | False | False | False | Occurs when both logRetention and logGroup are undefined and USE_CDK_MANAGED_LOGGROUP is false |
+| **USE_CDK_MANAGED_LOGGROUP** | CDK Lambda function construct creates log group with default props | True | True | True | Feature flag must be enabled |
```

Comment:
is prop injection available without escape hatch?

also prop injection meant custom configuration on the props right?